pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -20,7 +20,7 @@
 #define HTTP2_MAX_HEADERS_COUNT_FOR_PROCESSING 2
 
 // Maximum size for the path buffer.
-#define HTTP2_MAX_PATH_LEN 160
+#define HTTP2_MAX_PATH_LEN 120
```

Comment:
We need to restore this value, or to separate PRs, such a change cannot be introduced under http2 metrics PR

Reply:
At least locally, I was able to restore this value to 160 after applying my comments

Reply:
was able to increase it to 160 as well after the changes. really nice! thanks :)